apps/src/gamelab/GameLabJr.interpreted.js

```diff
@@ -256,68 +256,71 @@ function hideTitleScreen() {
 function draw() {
   background(World.background_color || "white");
 
-  // Run input events
-  for (var i=0; i<inputEvents.length; i++) {
-    const eventType = inputEvents[i].type;
-    const event = inputEvents[i].event;
-    const param = inputEvents[i].param;
-    if (eventType(param)) {
-      event();
+  if (World.frameCount > 1) {
```

Comment:
Is this still needed if we're only calling `drawSprites()` for predraw?

Reply:
I'm in favor of running the draw loop instead, but I made it an levelbuilder setting so we can try both. I'm going to leave this in unless/until we settle on using `drawSprites()` (or perhaps defining a `drawWorld()` to handle the actual rendering part of the draw loop and calling that instead).